perllib/FixMyStreet/Cobrand/Buckinghamshire.pm

```diff
@@ -145,6 +147,35 @@ sub send_questionnaires {
     return 0;
 }
 
+=head2 post_report_sent
+
+Bucks have a special 'Littering From Vehicles' category; any reports made in
+that category are automatically set to 'Investigating'.
+
+=cut
+
+sub post_report_sent {
+    my ( $self, $problem ) = @_;
+
+    if ( $problem->category eq VEHICLE_LITTERING_CATEGORY ) {
+        my $new_state = 'investigating';
+        $problem->update( { state => $new_state } );
+
+        my $template = $problem->response_template_for( $new_state, 'dummy' );
```

Comment:
I *think* that could work? But my main concern being getting Bucks people to understand why they (or we) have to change the template type from 'investigating' to 'confirmed', I don't know if that would cause them confusion? I.e. is it worth keeping more complex code if it keeps things conceptually simpler for the client?

I'll have a play with the code anyway; there could be something I'm missing.

Reply:
Ah yeah, that's much simpler, no faffing with another add_to_comments call.